test/honeyswap-default.test.js

```diff
@@ -105,30 +105,33 @@ describe("buildList", () => {
     for (let token of defaultTokenList.tokens) {
       let url = token.logoURI;
       // if tokenURI have that format: *1Hive/default-token-list/master/src/assets/* then replace to *1Hive/default-token-list/{branch}/src/assets*
-      if (url.includes(`https://raw.githubusercontent.com/1Hive/default-token-list/master/src/assets/gnosis/${token.address.toLowerCase()}/logo.`)) {
+      if (url.includes(`https://raw.githubusercontent.com/1Hive/default-token-list/`)) {
```

Comment:
Idr why i did that why, i think was about be able to locate logo in the CD/CI that not in master yet. Anyway if that its working that way lets that way and forget the auto branch update feature.